drivers/gpu/drm/rp1/rp1-dsi/rp1_dsi_dsi.c

```diff
@@ -1337,17 +1349,17 @@ static u32 get_colorcode(enum mipi_dsi_pixel_format fmt)
 }
 
 /* Maximum frequency for LP escape clock (20MHz), and some magic numbers */
-#define RP1DSI_ESC_CLK_KHZ      20000
+#define RP1DSI_ESC_CLK_MAX   20000000
 #define RP1DSI_TO_CLK_DIV           5
-#define RP1DSI_HSTX_TO_MIN      0x200
 #define RP1DSI_LPRX_TO_VAL      0x400
 #define RP1DSI_BTA_TO_VAL       0xd00
 
 void rp1dsi_dsi_setup(struct rp1_dsi *dsi, struct drm_display_mode const *mode)
 {
 	u32 timeout, mask, vid_mode_cfg;
-	int lane_kbps;
 	unsigned int bpp = mipi_dsi_pixel_format_to_bpp(dsi->display_format);
+	u32 byte_clock = clamp((bpp * 125u * (u32)min(mode->clock, 200000)) / dsi->lanes,
+			       10000000u, 187500000u);
```

Comment:
Magic numbers,

Reply:
Min and max DSI rates in bytes/second. I can add them to the #defines above.

Reply:
I would, just to make it clear.

Reply:
Done.